ADALiOS/ADALiOS/ADTokenCacheStoreItem.h

```diff
@@ -46,6 +46,13 @@
 
 @property (retain) ADUserInformation* userInformation;
 
+/*! Correlation ID of the request through which we get the token.
+ If the token is tombstoned, the value will be the correlation ID of the request that we got the error from.  */
+@property (retain) NSString* correlationId;
+
+/* Whether the token is tombstoned. */
+@property BOOL markAsDead;
```

Comment:
This is a verb phrase, a property should be a noun, preferably just "tombstone."